scripts/create_helm_charts_pr.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This script is used to open a PR against this repository:
+# https://github.com/SUSE/kubernetes-charts-suse-com
+# It will fetch the bundle specified by CAP_BUNDLE and will update the helm
+# charts in the above repo.
+
+set -e
+
+if [ -z "$GITHUB_USER"  ]; then
+  echo "GITHUB_USER environment variable not set"
+  exit 1
+fi
+
+if [ -z "$GITHUB_PASSWORD" ]; then
+  echo "GITHUB_PASSWORD environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${CAP_BUNDLE}" ]]; then
+  echo "CAP_BUNDLE not set"
```

Comment:
We should probably rename this, because it isn't `CAP`.

Reply:
+1

Reply:
Fixed in 57ffa5d2169a8e3ffa03b9a4923c641c6e18ef08